test/crowdsale/PausableCrowdsale.test.js

```diff
@@ -0,0 +1,153 @@
+const shouldFail = require('../helpers/shouldFail');
+const expectEvent = require('../helpers/expectEvent');
+
+const PausableCrowdsale = artifacts.require('PausableCrowdsaleImpl');
+const SimpleToken = artifacts.require('SimpleToken');
+
+require('../helpers/setup');
+
+contract('PausableCrowdsale', function ([owner, wallet, stranger]) {
+  const rate = 1;
+  const value = 1;
+
+  beforeEach('setting up', async function () {
```

Comment:
```suggestion
  beforeEach(async function () {
```

I wouldn't add these until #1260 is addressed, to keep everything consistent. Your call.